sql/core/src/test/resources/sql-tests/inputs/pgSQL/date.sql

```diff
@@ -228,9 +228,9 @@ SELECT f1 - date '2000-01-01' AS `Days From 2K` FROM DATE_TBL;
 -- test extract!
 --
 -- epoch
---
```

Comment:
Let's not remove this. This is the original comment.

Reply:
reverted